posthog/hogql_queries/web_analytics/stats_table.py

```diff
@@ -149,7 +149,7 @@ def to_path_scroll_bounce_query(self) -> ast.SelectQuery:
             min(session.$start_timestamp ) AS start_timestamp
         FROM events
         WHERE and(
-            events.event == '$pageview',
+            or(events.event == '$pageview', events.event == '$screen'),
```

Comment:
Can we add a virtual/computed field to the events Clickhouse table to aid with this? Something akin to a boolean `is_pageview_like`?

Or, maybe, something we could add to the HogQL translation layer instead? I believe mixing these will become cumbersome